src/app/view/IAP/hooks/useClouderyOffer.ts

```diff
@@ -94,9 +97,20 @@ export const useClouderyOffer = (): ClouderyOfferState => {
       }
 
       const doAsync = async (): Promise<void> => {
-        await initConnection()
-        await getSubscriptions({ skus: SKUS })
-        setPartialPopupUrl(current => current ?? href)
+        try {
+          await initConnection()
+          await getSubscriptions({ skus: SKUS })
+          setPartialPopupUrl(current => current ?? href)
+        } catch (error: unknown) {
+          const errorMessage = getErrorMessage(error)
```

Comment:
:o I finally understood the purpose of `getErrorMessage`. Will use it in backup feature...